src/test/java/org/minions/devfund/alejandro/movie/database/MovieDatabaseTests.java

```diff
@@ -0,0 +1,7 @@
+package org.minions.devfund.alejandro.movie.database;
+
+/**
+ * This class contain all the tests for MovieDatabase class.
+ */
+public class MovieDatabaseTests {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty class, write its code or make it an "interface". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2094)